Samples/iOS/Swift/IOS-SDK-Demo-App/Ad Formats/LOInlineViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  LOInlineViewController.swift
+//  PublicSampleAppTrial
+//
+
+import UIKit
+import VungleAdsSDK
+
+class LOInlineViewController: UIViewController {
+
+    @IBOutlet weak var tableView: UITableView!
+
+    var placementId: String?
+    var adView: VungleBannerView?
+    var models = [InlineModel].init(repeating: InlineModel(), count: 10)
+    let adCell = InlineAdCell(style: .default, reuseIdentifier: "adCell")
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationItem.title = "Liftoff Inline Ad"
+        if let placementId = placementId {
+            adView = VungleBannerView(placementId: placementId, vungleAdSize: .VungleAdSizeFromCGSize(CGSize(width: 300, height: 300)))
+            adView?.delegate = self
+            adView?.load()
+        }
+    }
+
+    @IBAction func btnPressed(_ sender:UIButton) {
+    }
+}
+
+extension LOInlineViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return models.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let model = models[indexPath.row]
+        if model.isAd {
+            return adCell
+        }
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "InlineCell", for: indexPath) as? InlineCell else {
+            return UITableViewCell()
+        }
+        return cell
+    }
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return models[indexPath.row].cellHeight
+    }
+}
+
+extension LOInlineViewController: VungleBannerViewDelegate {
+    func bannerAdDidLoad(_ bannerView: VungleAdsSDK.VungleBannerView) {
+        var adModel = InlineModel()
+        adModel.isAd = true
+        adModel.cellHeight = 300
+        models.insert(adModel, at: 1)
+        adCell.contentView.addSubview(bannerView)
+        tableView.reloadData()
+    }
+
+    func bannerAdWillPresent(_ bannerView: VungleAdsSDK.VungleBannerView) {
+        print("bannerAdWillPresent")
```

Comment:
Ideally, you need to add callbacks to `tableView` as we did for other ad types? 
No need to update with this PR. but for future improvement.